src/gluonts/shell/serve/__init__.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# Standard library imports
+# import signal
+from typing import Optional
+
+# Third-party imports
+import flask
+from gunicorn.app.base import BaseApplication
+from pydantic import BaseSettings
+
+# First-party imports
+from gluonts.core.component import check_gpu_support
+
+from gluonts.model.predictor import Predictor
+from gluonts.shell.env import SageMakerEnv
```

Comment:
I cannot locate the `SageMakerEnv` definition in this diff. Shouldn't this be part of the same PR?